ADOPTERS.md

```diff
@@ -34,7 +34,7 @@ This list is sorted in the order that organizations were added to it.
 | [Nethopper](https://nethopper.io) | @ddonahuex | Main IaC component in Nethopper's [Cloud Management Platform](https://www.nethopper.io/platform). Nethopper's Cloud Management Platform combines Crossplane with Continuous Delivery to allow DevOps to create, update, and destroy infrastructure in any cloud.|
 | [Renault](https://www.renaultgroup.com/) | @smileisak | Building Renault Kubernetes Platform resources using XRDs and compositions for an additional layer of abstraction to provide end-user services. |
 | [Wellhub (formerly Gympass)](https://wellhub.com) | @caiofralmeida @LCaparelli | Builds a self-service platform so engineers can be more productive in resource provisioning. |
-| [Deutsche Kreditbank AG](https://www.dkb.de/) | @akesser | At DKB, we have fully integrated Crossplane into our DKB Standard Operating Platform. Starting to move to production in mid-2023, all our clusters now use Crossplane to manage thousands of resources. |
+| [Deutsche Kreditbank AG](https://www.dkb.de/) | @TheBelgarion| At DKB, we have fully integrated Crossplane into our DKB Standard Operating Platform. Managing more then 10++ EKs clusters with Crossplane to manage thousands of resources. We did switch from crossplane to upbound provider for most of the main providers|
```

Comment:
should this be `more than 10+ EKS clusters`?

Reply:
also, I assume this is running in production now, right? the last statement from mid-2023 was that it was starting to move to production, so if it is in production now, can we explicitly state that? what do you think of this wording:

> Managing more than 10+ EKS clusters in production environments with Crossplane...